docs/CHANGES.md

```
@@ -1,381 +1,411 @@
+=== 3.0.0.001@2025-07-10 ===
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)